src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
+  --hover-color: #00acdc;
   margin: 0;
 }
+
+* {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
don't apply this styles for all elements on the page, leave only box-sizing under this selector